frontend/packages/knative-plugin/src/components/knative-resource/KSRoutesOverviewList.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import { ListGroup } from 'patternfly-react';
+
+import { K8sResourceKind, referenceForModel } from '@console/internal/module/k8s';
+import { ResourceLink, ExternalLink } from '@console/internal/components/utils';
+import { RouteModel } from '@console/knative-plugin';
+
+type KSRoutesOverviewListItemProps = {
+  route: K8sResourceKind;
+};
+
+type KSRoutesOverviewListProps = {
+  ksroutes: K8sResourceKind[];
+};
+
+const KSRoutesOverviewListItem: React.FC<KSRoutesOverviewListItemProps> = ({
+  route: {
+    metadata: { name, namespace },
+    status: { url },
+  },
+}) => {
+  return (
+    <li className="list-group-item">
+      <ResourceLink kind={referenceForModel(RouteModel)} name={name} namespace={namespace} />
+      <span className="text-muted">Location: </span>
+      <ExternalLink href={url} additionalClassName="co-external-link--block" text={url} />
+    </li>
+  );
+};
+
+const KSRoutesOverviewList: React.FC<KSRoutesOverviewListProps> = ({ ksroutes }) => (
+  <ListGroup componentClass="ul">
+    {_.map(ksroutes, (route) => (
+      <KSRoutesOverviewListItem key={route.metadata.uid} route={route} />
+    ))}
+  </ListGroup>
+);
+
+export default KSRoutesOverviewList;
```

Comment:
This is in the `knative-plugin` package. I don't think you need to call this `KSRoutes...`. Drop the `KS` prefix?